### DISCUSSION TITLE
duration 60 === 86 seconds on Android

### DISCUSSION THREAD

**Discussion by prechapl** [vydimitrov/react-countdown-circle-timer#164] [Q&A]
Hi there,

Thanks for creating this package.  I'm working on RN app where the timer is much slower on Android than on IOS. 

setting duration 60 === 86 seconds on Android

setting duration 60 === 63 to 67 seconds on IOS

Any advice on how to optimize this would be greatly appreciated!

Thanks,
Preston

**Comment by vydimitrov** [comment]
Also a bug is opened on this - https://github.com/vydimitrov/react-countdown-circle-timer/issues/165

**Comment by prechapl** [comment]
Yes, sorry. I tried to delete this discussion after creating the bug but couldn't.
